### PR TITLE
Use `termImg.string` as the default function

### LINKED ISSUE #12: Use `termImg.string` as the default function
For parity with https://github.com/sindresorhus/terminal-image, term-img, should only return the image as a string to be manually logged.

cc @sindresorhus

### PR DESCRIPTION
**Pull request by Richienb**
Fixes: #12 

cc @sindresorhus

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /// <reference types="node"/>
-import {ImageOptions} from 'ansi-escapes';
+import { ImageOptions } from 'ansi-escapes';
```
Comment (sindresorhus) [inline index.d.ts]: Don't do unrelated changes.

Comment (Richienb) [reply index.d.ts]: Fixed!
